dev/README_RELEASE_PROVIDER_PACKAGES.md

```diff
@@ -414,6 +414,16 @@ If you have providers as list of provider ids because you just released them you
 ```shell script
 cd "${AIRFLOW_REPO_ROOT}"
 
+breeze release-management publish-docs providers-index amazon cncf.kubernetes --override-versioned --run-in-parallel
+
+breeze release-management add-back-references all-providers
```

Comment:
mmm I guess this is not right here?
in this part we want to release only some providers
so I guess I should change this to

`breeze release-management add-back-references amazon cncf.kubernetes`

to followup the above publish-docs one?

Reply:
Oh yes on careful observation it should be what you mentioned. But I thought it was a general template. Let us make it the way you said

Reply:
see https://github.com/apache/airflow/pull/34295/commits/760681f03a947656f8faf2440d2b0e1176e40a33

Reply:
Changes are fine but did we get the short hand options working for add back references command? I'm wondering that right now. Afk, or would've checked